### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by umgefahren**
Introduced new license

### CODE REVIEW

**Review by tectrixer — APPROVED** [review]
Nice, I like MIT

**Review by Zollerboy1 — APPROVED** [review]

**Comment by vypxl** [comment]
I would like to be mentioned in the license. Please add all contributors separately or say "& contributors"

**Comment by Zollerboy1** [comment]
Maybe a separate Contributors.txt file?

**Review by vypxl — APPROVED** [review]

File: CONTRIBUTERS.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+vypxl
+Zollerboy1
+tectrixer
```
Comment (tectrixer) [inline CONTRIBUTERS.txt:3]: Please spell TecTrixer

**Review by tectrixer — CHANGES_REQUESTED** [review]
Please change my name, see single line comment

**Pull request merged**